test_requirements.txt

```diff
@@ -3,5 +3,5 @@ Pillow
 pytest
 xlrd
 numpy
-netCDF4==1.1.1
+netCDF4
```

Comment:
We *did* discover that this is something that needs to be updated, but I'd be inclined to not include it in this PR, since it's not really related to a "README update".